metals/src/main/scala/scala/meta/internal/metals/PackageProvider.scala

```diff
@@ -0,0 +1,41 @@
+package scala.meta.internal.metals
+
+import org.eclipse.lsp4j.Position
+import org.eclipse.lsp4j.Range
+import org.eclipse.lsp4j.TextEdit
+import org.eclipse.lsp4j.WorkspaceEdit
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+class PackageProvider(private val buildTargets: BuildTargets) {
+
+  def addPackageWorkspaceEdit(path: AbsolutePath): Option[WorkspaceEdit] = {
+    def createWorkspaceEdit(packageName: String): WorkspaceEdit = {
```

Comment:
This can be a private def too, it's weird to have one private and the other local.